osu.Game/Online/ScoreBasedUserStatisticsUpdate.cs

```diff
@@ -9,7 +9,7 @@ namespace osu.Game.Online
     /// <summary>
     /// Contains data about the change in a user's profile statistics after completing a score.
     /// </summary>
-    public class UserStatisticsUpdate
+    public class ScoreBasedUserStatisticsUpdate
```

Comment:
This is a bit on the iffy side and it may not sit well with everyone, but with `LocalUserStatisticsProvider.StatisticsUpdate` requiring a model to contain change to user statistics on a specific ruleset, this class needed to be renamed to something else. The other option would be to pick a different name for the `StatisticsUpdate` model, but I think renaming this model is better since it is focused on scores anyway.